core/state/statedb.go

```diff
@@ -995,10 +997,7 @@ func (s *StateDB) copyInternal(doPrefetch bool) *StateDB {
 		hasher:    crypto.NewKeccakState(),
 
 		// state expiry copy
-		epoch:             s.epoch,
-		enableStateExpiry: s.enableStateExpiry,
-		fullStateDB:       s.fullStateDB,
-		originalHash:      s.originalHash,
+		expiryMeta: s.expiryMeta,
```

Comment:
It may be better to do deep copy. Perhaps can add a `copy()` function to the expiryMeta struct.

Reply:
it's ok for expiryMeta here, it's a read-only instance, no concurrent problem here.